### DISCUSSION TITLE
Nodejs polyfills: Other packages —— fs net tls child_process, still prompt that the model cannot be found

### DISCUSSION THREAD

**Discussion by jennyluo0607** [quasarframework/quasar#10836] [CLI - SSR mode]
In order to display Echarts on the server side, I installed the echarts-ssr package. After installation, I got a mistake. I found that this is related to Nodejs polyfills, but some packages still don’t work. How to solve it?
![image](https://user-images.githubusercontent.com/87688455/135035852-3606290b-ab65-4cfe-aa7f-d5d0d4f9f782.png)

**Comment by TobyMosque** [comment]
https://quasar.dev/quasar-cli/developing-electron-apps/electron-preload-script

instead of directly import the NodeJS APIs (or any libs who use them) in the renderer process, you would expose then in the preload script.

**Reply by jennyluo0607** [reply]
Sorry, I chose the wrong discussion category, it should belong to SSR mode.
